tests/reg_tests/test_DVConstraints.py

```diff
@@ -807,132 +881,8 @@ def setUp(self):
-        handler.root_add_dict("derivs_deformed", funcsSens, rtol=1e-6, atol=1e-6)
-        return funcs, funcsSens
-
-    @unittest.skipIf(missing_geograd, "requires geograd")
-    def test_triangulatedSurface(self, train=False, refDeriv=False):
-        refFile = os.path.join(self.base_path, "ref/test_DVConstraints_triangulatedSurface.ref")
-        with BaseRegTest(refFile, train=train) as handler:
-            meshfile = os.path.join(self.base_path, "../../input_files/bwb.stl")
-            objfile = os.path.join(self.base_path, "../../input_files/blob_bwb_wing.stl")
-            ffdfile = os.path.join(self.base_path, "../../input_files/bwb.xyz")
-            testmesh = mesh.Mesh.from_file(meshfile)
-            testobj = mesh.Mesh.from_file(objfile)
-            # test mesh dim 0 is triangle index
-            # dim 1 is each vertex of the triangle
-            # dim 2 is x, y, z dimension
-
-            # create a DVGeo object with a few local thickness variables
-            DVGeo = DVGeometry(ffdfile)
-            nRefAxPts = DVGeo.addRefAxis("wing", xFraction=0.25, alignIndex="k")
-            self.nTwist = nRefAxPts - 1
-
-            def twist(val, geo):
-                for i in range(1, nRefAxPts):
-                    geo.rot_z["wing"].coef[i] = val[i - 1]
-
-            DVGeo.addGlobalDV(dvName="twist", value=[0] * self.nTwist, func=twist, lower=-10, upper=10, scale=1)
-            DVGeo.addLocalDV("local", lower=-0.5, upper=0.5, axis="y", scale=1)
-
-            # create a DVConstraints object for the wing
-            DVCon = DVConstraints()
-            DVCon.setDVGeo(DVGeo)
-            p0 = testmesh.vectors[:, 0, :]
-            v1 = testmesh.vectors[:, 1, :] - p0
-            v2 = testmesh.vectors[:, 2, :] - p0
-            DVCon.setSurface([p0, v1, v2], addToDVGeo=True)
-            p0b = testobj.vectors[:, 0, :]
-            v1b = testobj.vectors[:, 1, :] - p0b
-            v2b = testobj.vectors[:, 2, :] - p0b
-            DVCon.setSurface([p0b, v1b, v2b], name="blob")
-
-            DVCon.addTriangulatedSurfaceConstraint("default", "default", "blob", None, rho=10.0, addToPyOpt=True)
-            DVCon.addTriangulatedSurfaceConstraint("default", "default", "blob", None, rho=1000.0, addToPyOpt=True)
-
-            funcs, funcsSens = generic_test_base(DVGeo, DVCon, handler, fdstep=1e-3)
-            handler.assert_allclose(
-                funcs["DVCon1_trisurf_constraint_0_KS"], 0.34660627481696404, name="KS", rtol=1e-7, atol=1e-7
-            )
-            handler.assert_allclose(funcs["DVCon1_trisurf_constraint_0_perim"], 0.0, name="perim", rtol=1e-7, atol=1e-7)
-            funcs, funcsSens = self.bwb_test_twist(DVGeo, DVCon, handler)
-            funcs, funcsSens = self.bwb_test_deformed(DVGeo, DVCon, handler)
-
-    @unittest.skipIf(missing_geograd, "requires geograd")
-    def test_triangulatedSurface_intersected(self, train=False, refDeriv=False):
```

Comment:
Where did this end up?

Reply:
That is [here](https://github.com/mdolab/pygeo/blob/ab236b3eb91586d6fe23278010c13cf0b220a7ec/tests/reg_tests/test_DVConstraints.py#L799). I initially moved all the geograd tests into a separate class but realized I only needed to move the one with 2 DVGeos out.